src/Microsoft.ML.Onnx/SaveOnnxCommand.cs

```diff
@@ -200,13 +200,13 @@ private void Run(IChannel ch)
 
             if (_model == null)
             {
-                if (string.IsNullOrEmpty(Args.InputModelFile))
+                if (string.IsNullOrEmpty(ImplOptions.InputModelFile))
```

Comment:
>ImplOptions [](start = 41, length = 11)

If possible could you not use abbreviations? And here too, just curious why not `Options`?

Reply:
(plz see above)

---
In reply to: [258279957](https://github.com/dotnet/machinelearning/pull/2636#discussion_r258279957) [](ancestors = 258279957)